nxviz/plots.py

```diff
@@ -231,7 +240,12 @@ def compute_group_colors(self):
     def compute_edge_colors(self):
         """Compute the edge colors."""
         data = [self.graph.edges[n][self.edge_color] for n in self.edges]
-        data_reduced = sorted(list(set(data)))
+
+        if self.group_order == "alphabetically":
+            data_reduced = sorted(list(set(data)))
+        elif self.group_order == "default":
+            data_reduced = list(unique_everseen(data))
+
```

Comment:
This chunks looks repeated from lines 199-203. Perhaps it might be useful to have this pre-computed in the `__init__()`? It may help with maintainability going forward to have non-repeated chunks of code.

Reply:
Ah actually there it is a mistake on my side. It does not change the outcome but here in 245 the if statement is not needed. Sorry! I'll remove it. In line 199 we need it to have the right node colors according to the group_order!

Reply:
Ah okay! Got it!